### DISCUSSION TITLE
I am getting an error in terraform while creating instance.

### DISCUSSION THREAD

**Discussion by P-S-A-R** [learndevopsonline/learndevopsonline#410] [Q&A]
I am getting an error like unsuppoerted argument vps_security_id and also vps_security_ids too
![Screenshot 2023-04-04 194303](https://user-images.githubusercontent.com/125214818/229874922-8e511b60-4776-4bba-900e-0403d8804823.png)
![Screenshot 2023-04-04 194215](https://user-images.githubusercontent.com/125214818/229874923-daa4e29d-7e20-45a0-8ba9-9d50ed5097e8.png)

**Comment by noolivara** [comment]
@P-S-A-R,  Terraform command is failing due to an unsupported argument named "vps_security_group_id". 
You may have to use "vpc_security_group_ids" instead.

"vps_security_group_id" may not be a valid argument.

**Reply by P-S-A-R** [reply]
Its saying incorrect attribute value with vpc_security_group_id . I also tried with vps_security_group_ids
![Screenshot 2023-04-04 231524](https://user-images.githubusercontent.com/125214818/229924312-481059a4-f5c9-490b-af38-99e805964ac1.png)

**Comment by noolivara** [comment]
@P-S-A-R , The vpc_security_group_ids attribute expects a set of strings (i.e., a list of string values).

**Reply by P-S-A-R** [reply]
Thank you

**Comment by P-S-A-R** [comment]
Thank you very much . I realised my mistake i gave an extra space before closing the braces.
